pilot/pkg/networking/core/v1alpha3/listener_inbound.go

```diff
@@ -329,6 +361,20 @@ func (lb *ListenerBuilder) buildInboundChainConfigs() []inboundChainConfig {
 			if cc.bind == "" {
 				// If user didn't provide, pick one based on IP
 				cc.bind = getSidecarInboundBindIP(lb.node)
+				// if no global unicast address
+				if cc.bind == WildcardAddress || cc.bind == WildcardIPv6Address {
+					oWildcardAndLocalHost := NewWildcardAndLocalHost(lb.node.GetIPMode())
+					if oWildcardAndLocalHost == nil {
+						log.Warnf("inboundVirtualListener: Can not fetch wildcar and localhost from proxy [%s]", lb.node.ID)
```

Comment:
No need for this as mentioned previously

Reply:
done